http_post.go

```diff
@@ -2,21 +2,24 @@ package kapacitor
 
 import (
 	"encoding/json"
+	"fmt"
 	"log"
 	"net/http"
 	"sync"
 
 	"github.com/influxdata/kapacitor/bufpool"
 	"github.com/influxdata/kapacitor/models"
 	"github.com/influxdata/kapacitor/pipeline"
+	"github.com/influxdata/kapacitor/services/httppost"
 )
 
 type HTTPPostNode struct {
 	node
-	c   *pipeline.HTTPPostNode
-	url string
-	mu  sync.RWMutex
-	bp  *bufpool.Pool
+	c        *pipeline.HTTPPostNode
+	handlers []*httppost.Handler
```

Comment:
It feels odd that the HTTPPost node is dealing with alert handlers. Maybe we should rename the type  `httppost.Handler` to `httppost.Endpoint` since all it really is is an endpoint. This change also helps clarify the purpose of the type since currently the Handler type is not used for its `Handle` method, in this context.